src/components/navbar/Navbar.vue

```diff
@@ -168,10 +168,13 @@ export default {
             ])
         },
         genNavbarBrandNode() {
+            const children = this.$slots.brand != null
+                ? [this.$slots.brand(), this.genBurgerNode()]
+                : this.genBurgerNode()
```

Comment:
@wesdevpro I believe the updated code is legible enough and aligns with the style of the other part of the codebase.